internal/v1/server_test.go

```diff
@@ -911,101 +954,197 @@ func TestComposeCustomizations(t *testing.T) {
+				Customizations: &Customizations{
+					Packages: nil,
+				},
+				Distribution: "rhel-8",
+				ImageRequests: []ImageRequest{
+					{
+						Architecture: "x86_64",
+						ImageType:    ImageTypes_edge_commit,
+						Ostree: &OSTree{
+							Ref: strptr("edge/ref"),
+						},
+						UploadRequest: UploadRequest{
+							Type:    UploadTypes_aws_s3,
+							Options: AWSS3UploadRequestOptions{},
+						},
+					},
+				},
 			},
-			Distribution: "rhel-8",
-			ImageRequests: []ImageRequest{
-				{
+			composerRequest: composer.ComposeRequest{
+				Distribution: "rhel-8",
+				Customizations: &composer.Customizations{
+					Packages: nil,
+				},
+				ImageRequest: &composer.ImageRequest{
 					Architecture: "x86_64",
-					ImageType:    ImageTypes_edge_commit,
-					Ostree: &OSTree{
+					ImageType:    composer.ImageTypes_edge_commit,
+					Ostree: &composer.OSTree{
 						Ref: strptr("edge/ref"),
 					},
-					UploadRequest: UploadRequest{
-						Type:    UploadTypes_aws_s3,
-						Options: AWSS3UploadRequestOptions{},
+					Repositories: nil,
+					UploadOptions: &composer.UploadOptions{
+						Region: "",
 					},
 				},
 			},
 		},
 		{
-			Customizations: &Customizations{
-				Packages: &[]string{"pkg"},
-				Subscription: &Subscription{
-					Organization: 000,
+			imageBuilderRequest: ComposeRequest{
+				Customizations: &Customizations{
+					Packages: &[]string{"pkg"},
+					Subscription: &Subscription{
+						Organization: 000,
+					},
+				},
+				Distribution: "centos-8",
+				ImageRequests: []ImageRequest{
+					{
+						Architecture: "x86_64",
+						ImageType:    ImageTypes_rhel_edge_commit,
+						Ostree: &OSTree{
+							Ref:    strptr("test/edge/ref"),
+							Url:    strptr("https://ostree.srv/"),
+							Parent: strptr("test/edge/ref2"),
+						},
+						UploadRequest: UploadRequest{
+							Type:    UploadTypes_aws_s3,
+							Options: AWSS3UploadRequestOptions{},
+						},
+					},
 				},
 			},
-			Distribution: "centos-8",
-			ImageRequests: []ImageRequest{
-				{
+			composerRequest: composer.ComposeRequest{
+				Distribution: "centos-8",
+				Customizations: &composer.Customizations{
+					Packages: &[]string{
+						"pkg",
+					},
+					Subscription: &composer.Subscription{
+						ActivationKey: "",
+						BaseUrl:       "",
+						Insights:      false,
+						Organization:  "000",
+						ServerUrl:     "",
+					},
+				},
+				ImageRequest: &composer.ImageRequest{
 					Architecture: "x86_64",
-					ImageType:    ImageTypes_rhel_edge_commit,
-					Ostree: &OSTree{
+					ImageType:    composer.ImageTypes_edge_commit,
+					Ostree: &composer.OSTree{
 						Ref:    strptr("test/edge/ref"),
 						Url:    strptr("https://ostree.srv/"),
 						Parent: strptr("test/edge/ref2"),
 					},
-					UploadRequest: UploadRequest{
-						Type:    UploadTypes_aws_s3,
-						Options: AWSS3UploadRequestOptions{},
-					},
+					Repositories:  nil,
+					UploadOptions: &composer.UploadOptions{},
 				},
 			},
 		},
 	}
```

Comment:
Can we add a case for user customization? And verify that IB adds the wheel group?